vm/chef/cookbooks/jenkins/attributes/default.rb

```diff
@@ -13,110 +13,3 @@
 # limitations under the License.
 
 default['jenkins']['packages'] = %w(jenkins git groovy subversion)
-
-# Default plugins to install
-default['jenkins']['plugins'] = %w(ace-editor
```

Comment:
Will this cause a regression for marketplace users if we're removing all these default plugins?

Reply:
As per chat conversation, removing the plugins will: 
* Avoid dependencies errors on dashboard
* Several plugins actually are not working due these dependencies errors. 
* Reduce maintenance as we have to handle someway the app dependencies
* Improve experience as user can pick only the plugins they want via UI. And UI even resolves better the plugin dependencies.